src/renderer/ipc/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { ipcRenderer, IpcRendererEvent } from "electron";
+import { areArgsUpdateAvailableFromMain, UpdateAvailableChannel, onCorrect, UpdateAvailableFromMain, BackchannelArg } from "../../common/ipc";
+import { Notifications, notificationsStore } from "../components/notifications";
+import { Button, ButtonPannel } from "../components/button";
+import { isMac } from "../../common/vars";
+import * as uuid from "uuid";
+
+function UpdateAvailableHandler(event: IpcRendererEvent, ...[backchannel, updateInfo]: UpdateAvailableFromMain): void {
+  const notificationId = uuid.v4();
+
+  function sendToBackchannel(data: BackchannelArg): void {
+    notificationsStore.remove(notificationId);
+    console.log("sending to backchanel", { backchannel, data });
+    ipcRenderer.send(backchannel, data);
+  }
+
+  function renderYesButtons() {
```

Comment:
Could be better to put `RenderYesButtons` name and use as `<ReactComponent/>` inside notification?

Reply:
Probably, good idea